api/lanes.py

```diff
@@ -859,20 +859,49 @@ def fetch_recommendations(self, _db):
             return metadata.recommendations
         return []
 
-    def apply_filters(self, _db, qu, work_model, facets, pagination, featured=False):
+    def apply_filters(self, _db, qu, facets, pagination, featured=False):
         if not self.recommendations:
             return None
 
-        if work_model != Work:
-            qu = qu.join(LicensePool.identifier)
+        qu = qu.join(LicensePool.identifier)
         qu = Work.from_identifiers(
             _db, self.recommendations, base_query=qu
         )
         return super(RecommendationLane, self).apply_filters(
-            _db, qu, work_model, facets, pagination, featured=featured)
+            _db, qu, facets, pagination, featured=featured
+        )
+
+
+class SeriesFacets(Facets):
+    """A custom Facets object for ordering a lane based on series."""
```

Comment:
`Facets.apply` has logic for setting DISTINCT and ORDER BY on a query, logic that was duplicated in SeriesLane and that broke when the core logic around ORDER BY changed. Making a custom `Facets` subclass lets us create custom joins on the query that are used only when we're applying the ORDER BY.